x-pack/legacy/plugins/rollup/server/routes/api/search.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { RequestHandler } from 'src/core/server';
+import { callWithRequestFactory } from '../../lib/call_with_request_factory';
+import { isEsError } from '../../lib/is_es_error';
+import { licensePreRoutingFactory } from '../../lib/license_pre_routing_factory';
+import { API_BASE_PATH } from '../../../common';
+import { RouteDependencies, ServerShim } from '../../types';
+
+export function registerSearchRoute(deps: RouteDependencies, legacy: ServerShim) {
+  const handler: RequestHandler<any, any, any> = async (ctx, request, response) => {
+    const callWithRequest = callWithRequestFactory(deps.elasticsearchService, request);
+    try {
+      const requests = request.body.map(({ index, query }: { index: string; query: any }) =>
+        callWithRequest('rollup.search', {
+          index,
+          rest_total_hits_as_int: true,
```

Comment:
Just as a side note - AFAIK `rest_total_hits_as_int` will get deprecated with 8.0, so this has to be handled differently in future.

Reply:
Thanks for the heads up! Do you know if there is an issue open somewhere to track this?

Reply:
This seems to be the main one: https://github.com/elastic/kibana/issues/26356